user_docs/en/userGuide.t2t

```diff
@@ -2865,6 +2874,13 @@ Specifically, following issues can be solved by running this tool:
 
 ++ Reload plugins ++[ReloadPlugins]
 This item, once activated, reloads app modules and global plugins without restarting NVDA, which can be useful for developers.
+You can also find out whether the application that you focus has an associated app module.
+
+%kc:beginInclude
+|| Name | Desktop key | Laptop key | Touch | Description |
+| Report loaded plugin and executable | ``NVDA+control+f1`` | ``NVDA+control+f1`` | None | Report the name of the app module associated to the current application, if any, and the name of the executable of the currently running application. |

```

Comment:
1. Replace plugin by appModule; it's more precise
2. Replace "currently running application" by "active application". Indeed, many applications are currently running but only one is active at a time (active in the meaning of "having the focus"). One could also say "foreground application", but this is maybe more developer oriented?

```suggestion
| Report loaded appModule and executable | ``NVDA+control+f1`` | ``NVDA+control+f1`` | None | Report the name of the app module associated to the current application, if any, and the name of the executable of the active application. |
```

Reply:
Slight fix, it should be "app module" with a space

```suggestion
| Report loaded app module and executable | ``NVDA+control+f1`` | ``NVDA+control+f1`` | None | Report the name of the app module associated to the current application, if any, and the name of the executable of the active application. |
```

Reply:
> 1. Replace plugin by appModule; it's more precise
> 
>     2. Replace "currently running application" by "active application". Indeed, many applications are currently running but only one is active at a time (active in the meaning of "having the focus"). One could also say "foreground application", but this is maybe more developer oriented?

I've changed this part to consider this.